src/upgrade/steps/plpgsql-resolve.js

```
@@ -0,0 +1,73 @@
+  const sql = "SELECT count(*) FROM pg_proc p JOIN pg_language l ON p.prolang = l.oid WHERE l.lanname = 'plpgsql' AND p.proowner != 10";
+  const out = pgQuery({ db, sql, captureStdout: true });
+  return parseInt(out, 10) > 0;
+}
+
+async function plan() {
+  let dbs;
+  try { dbs = listUserDbs(); } catch (err) { return `cannot enumerate DBs: ${err.message}`; }
+  return `would DROP+CREATE plpgsql in ${dbs.length} user DB(s): ${dbs.join(', ')} (skipping any with user-owned plpgsql functions)`;
+}
+
+async function execute({ log, warn }) {
```

ESLint

[error] 'log' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)